client/app/assets/json/currency_iso.json

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
This file is money gem's configuration.
I've only added some of the data points: name, subunit_to_unit, symbol, [disambiguate_symbol]